daeun/머쓱이보다-키-큰-사람.js

```diff
@@ -0,0 +1,6 @@
+function solution(array, height) {
+        let count = array.filter((v)=>{
+            return v > height
+        }).length
```

Comment:
```suggestion
        const tallStudent = array.filter(v => v > height).length;
```

- 한줄로 끝나는거라 축약문법 사용했고 변하지 않는 수이기에 const로 변경했습니다.
- 변수명 명확하게 변경했습니다~!

Reply:
@yunwoo-yu 한 줄로 끝나면 축약문법을 사용해줄 수 있군요.. 감사합니다!